packages/awslint/lib/case.ts

```diff
@@ -0,0 +1,18 @@
+import * as changeCase from 'change-case';
+
+function cachedTransform(transform: (x: string) => string): (x: string) => string {
```

Comment:
Throwing this in for good measure, so we never convert the same string twice. It saves 3-4s.